docs/developer/extending/webhooks/subscription-webhook-payloads.mdx

```diff
@@ -36,6 +36,63 @@ subscription {
 }
 ```
 
+### Filterable subscription query
+
+:::info
+
+This feature was introduced in **Saleor 3.20**.
+
+:::
+
+:::caution
+
+This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
+feedback. However, it is still undergoing development and is subject to modifications.
+
+:::
+
+For some webhook events, you can create a filterable subscription. A filterable subscription means the event will be delivered only when the filter input matches the object.
```

Comment:
I would also mention some use case to show when it's useful.

"This is useful if you want to receive webhooks only for selected channels or your shop is generating lots of events and you want to split them between different receivers."

Not sure if it makes sense, but something like this would be nice.